sensor_msgs/include/sensor_msgs/point_field_conversion.h

```diff
@@ -0,0 +1,146 @@
+/*
+ * Software License Agreement (BSD License)
+ *
+ * Robot Operating System code by the University of Osnabrück
+ * Copyright (C) 2015 University of Osnabrück
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in the
+ *    documentation and/or other materials provided with the distribution.
+ * 3. All advertising materials mentioning features or use of this software
```

Comment:
Right, it's obnoxious :) We usually choose the 3-clause one, cf http://wiki.ros.org/DevelopersGuide#Licensing